src/commands/manifest.ts

```diff
@@ -11,13 +11,25 @@ export default class Manifest extends Command {
     {name: 'path', description: 'path to plugin', default: '.'}
   ]
 
+  capturePluginWarning(warning: Error) {
+    if (warning.name.match(/Plugin:/)) { throw warning }
+  }
+
   async run() {
     try { fs.unlinkSync('.oclif.manifest.json') } catch {}
     const {args} = this.parse(Manifest)
     const root = path.resolve(args.path)
     let plugin = new Config.Plugin({root, type: 'core', ignoreManifest: true})
-    await plugin.load()
     if (!plugin) throw new Error('plugin not found')
```

Comment:
This should be done first, otherwise, you could get a "load on undefined" problem above.

Reply:
you're right, I'm curious why I had it the other way. Must just have been something that happened in a refactor I didn't intend